src/components/connect.tsx

```diff
@@ -622,18 +621,9 @@ function connect<
       const lastWrapperProps = useRef(wrapperProps)
       const childPropsFromStoreUpdate = useRef<unknown>()
       const renderIsScheduled = useRef(false)
-      const isProcessingDispatch = useRef(false)
```

Comment:
this one is a bonus removal - this ref stayed completely unused

Reply:
current-me has no clue what past-me was thinking for this variable :)

Reply:
Such is the live of a developer. What's comforting is that Future You thinks Present You is a moron, too. 🤣 

Of course this is a good thing. We want Future Us to be smarter than Present Us. 📈